Code/Editor/EditorModularViewportCameraComposer.cpp

```diff
@@ -145,13 +145,27 @@ namespace SandboxEditor
             }
         };
 
+        const auto trackingTransform = [viewportId = m_viewportId]
```

Comment:
q: I know its not needed but is there anything in our code standards about omitting () on lambdas?

Reply:
Hmm I don't think there is, I don't think it's a huge deal though right? I like omitting them if they're not needed, annoyingly they are if you need to add `-> <return type>` or `constexpr`